.cirrus.yml

```diff
@@ -37,13 +37,18 @@ task:
     - choco upgrade -y chocolatey
     - choco install -y --no-progress ant
     - choco install -y --no-progress openjdk --version %OPENJDK_VERSION%
-    # - choco install -y --no-progress xsltproc
-    # - choco install -y --no-progress xmlstarlet
+    - set "JAVA_HOME=%ProgramFiles%\OpenJDK\%OPENJDK_PATH%"
+    - set "PATH=%JAVA_HOME%\bin;%PATH%"
+    - java -version
+    - javac -version
+
   version_script:
+    - set "JAVA_HOME=%ProgramFiles%\OpenJDK\%OPENJDK_PATH%"
+    - set "PATH=%JAVA_HOME%\bin;%PATH%"
     - set
     - ant -version
     - java --version
-    - ./mvnw --version
+    - mvn --version
```

Comment:
Please remove this command at all, we use mvnw now, version of maven on host doesn't matter anymore.

Please add 
```
- set "JAVA_HOME=%ProgramFiles%\OpenJDK\%OPENJDK_PATH%"
    - set "PATH=%JAVA_HOME%\bin;%PATH%"
```

Before each `cd ci`